examples/jsm/loaders/SVGLoader.js

```diff
@@ -585,20 +595,27 @@ SVGLoader.prototype = Object.assign( Object.create( Loader.prototype ), {
 
 					case 'a':
 						var numbers = parseFloats( data );
-
 						for ( var j = 0, jl = numbers.length; j < jl; j += 7 ) {
+							// skip command if start point == end point
+							if( numbers[ j + 5 ] == 0 && numbers[ j + 6 ] == 0 ) continue
```

Comment:
Shouldn't this line be ```if( numbers[ j + 5 ] == point.x && numbers[ j + 6 ] == point.y ) continue``` ?

Reply:
Νο, the 'a' indicates that the arguments are relative to the current point, so a 0 means no displacement

Reply:
Ok, missed that